pcsx2/GS/Renderers/HW/GSTextureCache.cpp

```diff
@@ -196,21 +192,11 @@ GSTextureCache::Source* GSTextureCache::LookupDepthSource(const GIFRegTEX0& TEX0
 		// Note: might worth to check previous frame
 		// Note: otherwise return NULL and skip the draw
 
-		if (m_renderer->m_game.title == CRC::JackieChanAdv || m_renderer->m_game.title == CRC::SVCChaos)
-		{
-			// JackieChan and SVCChaos cause regressions when skipping the draw calls so we reuse the old code for these two.
-			return LookupSource(TEX0, TEXA, r, false);
-		}
-		else
-		{
-			// Full Spectrum Warrior: first draw call of cut-scene rendering
-			// The game tries to emulate a texture shuffle with an old depth buffer
-			// (don't exists yet for us due to the cache)
-			// Rendering is nicer (less garbage) if we skip the draw call.
-			throw GSRecoverableError();
-		}
-
-		//ASSERT(0);
+		// Full Spectrum Warrior: first draw call of cut-scene rendering
+		// The game tries to emulate a texture shuffle with an old depth buffer
+		// (don't exists yet for us due to the cache)
+		// Rendering is nicer (less garbage) if we skip the draw call.
```

Comment:
The comment is a bit misleading, this suggest that this code path is only for Full Spectrum Warrior, maybe make it a bit more generic.

Reply:
And completely outside the scope of this PR, this PR isn't a "lets critique the texture cache" PR, it's removing a CRC hack.  If you have problems with other parts of the file, make a new issue.

Reply:
Okay okay, this isn't meant to be free "critique", sorry if you felt it was the case, that's fine then.